packages/at_commons/lib/src/keystore/at_key.dart

```diff
@@ -347,11 +347,6 @@ class PublicKey extends AtKey {
     super.metadata = Metadata();
     super.metadata.isPublic = true;
   }
-
-  @override
-  String toString() {
-    return 'public:$key${_dotNamespaceIfPresent()}$_sharedBy'.toLowerCase();
-  }
```

Comment:
Can all of the other subclasses' toString methods be removed as well?

Reply:
I will verify this by writing tests.

Reply:
Verified using unit tests before and after removing the toString